azure/src/index.js

```diff
@@ -267,6 +271,12 @@ const processNSGLogData = (nsgLogs) => {
                                             processedLog[flowPropertiesName[l]] = timestamp || tuples[l];
                                             continue;
                                         }
+                                        //Adding legends full name
+                                        if (legendsName[flowPropertiesName[l]]) {
+                                            processedLog[flowPropertiesName[l]] =
+                                                legendsName[flowPropertiesName[l]][tuples[l]];
+                                            continue;
+                                        }
```

Comment:
I see this is duplicate code in all 3 files. Is it possible to keep it in one place and re-use from there? It avoid making changes in all the places